serverless/installing_serverless/upgrading-serverless.adoc

```diff
@@ -0,0 +1,10 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="upgrading-serverless"]
+= Upgrading {ServerlessProductName}
+include::modules/common-attributes.adoc[]
+:context: upgrading-serverless
+toc::[]
+
+If you have previously installed version of {ServerlessProductName}, follow the instructions in this guide to upgrade to the latest version.
```

Comment:
Missing an "a" here (e.g. s/If you have previously installed version/If you have installed a previous version)